elements/drop.down.element.js

```diff
@@ -1,6 +1,15 @@
 let BaseElement = require("./base.element");
 
 class DropDown extends BaseElement {
+    async open() {
+        console.log(`Open drop-down ${this.elementName}`);
+        await this.protractorElement.click();
+    }
+
+    async selectItem(itemName) {
+        this.protractorElement = `//option[contains(., ${itemName})]`;
+        await element(by.xpath(this.protractorElement)).click();
```

Comment:
Crate `new Button(...` and use it instead of `element(by.xpath(this.protractorElement))`